modules/cli/src/main/scala/scala/cli/commands/run/Run.scala

```diff
@@ -570,7 +570,14 @@ object Run extends ScalaCommand[RunOptions] with BuildCommandHelpers {
       case Platform.JVM =>
         runMode match {
           case RunMode.Default =>
+            val sources = builds.head.inputs.sourceFiles().map {
+              case s: ScalaFile => s.path.toString.replace('\\', '/')
+              case s: Script    => s.path.toString.replace('\\', '/')
+              case _            => ""
+            }.filter(_.nonEmpty).distinct.mkString("|")
```

Comment:
```suggestion
            }.filter(_.nonEmpty).distinct.mkString(File.pathSeparator)
```
I wonder if it wouldn't be less surprising to use the path separator here, rather than `|`